TowerForge/TowerForge/GameModule/Events/ConcurrentEvent.swift

```diff
@@ -11,8 +11,8 @@ struct ConcurrentEvent: TFEvent {
     var timestamp: TimeInterval
     var entityId: UUID
 
-    private let event1: TFEvent
-    private let event2: TFEvent
+    var event1: TFEvent
+    var event2: TFEvent
```

Comment:
Is there a  need to expose the events if we already have a method to create a new concurrent event with transforms applied?

Reply:
Updated – this was from another method that I attempted.